webpack.mix.js

```diff
@@ -11,7 +11,11 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-    .postCss('resources/css/app.css', 'public/css', [
+mix.js("resources/js/app.js", "public/js").postCss(
+    "resources/css/app.css",
+    "public/css",
+    [
         //
-    ]);
+    ]
+);
+mix.browserSync("127.0.0.1:8000");
```

Comment:
This looks good but you should avoid hardcoding the port and host because your teammate might be serving the application using a different host and port. basically, those values are kept in the `.env` file